src/components/kytos/tabs/tabs.vue

```diff
@@ -58,9 +58,29 @@ export default {
       content: {},
       notifications: [],
       hiddenPanel: true,
+      lastTab: -1
     }
   },
+  watch: {
+    hiddenTabs: function () {
+      /** Watch the hiddenTabs and get all elements with class="tab-nav"
+       * to remove the class "active" when tabs are closed,
+       * and reactive the last used tab
+       **/
+      let tablinks = document.getElementsByClassName("tab-nav");
+      if (this.hiddenTabs) {
+        for (let i = 0; i < tablinks.length; i++) {
+          tablinks[i].className = tablinks[i].className.replace(" active", "")
```

Comment:
I think there's a more elegant way to add/remove classes to an element - `.toggle()` maybe?.

Reply:
Yes, I think so, I'll do the change! haha

Reply:
I didn't notice before that this is not a jquery operation, so there is no method toggle for className because this is a string

Reply:
I changed the tablinks to jquery objects, I think that's better now.